Sources/LLVM/TokenType.swift

```diff
@@ -8,6 +8,11 @@ import cllvm
 public struct TokenType: IRType {
   internal let llvm: LLVMTypeRef
 
+  internal init() {
+    // FIXME: This is just plain wrong.
+    self.llvm = PointerType(pointee: VoidType()).asLLVM()
```

Comment:
I actually have no idea what to do about this.  `@llvm.coro.id` is capable of producing `Token`s that are consumed by the other `@llvm.coro.*` functions so we clearly need some way to initialize this properly, but there's nothing I can find in the C API that will do it.